pkg/util/varz/varz.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package varz
```

Comment:
Note we *always* expose debug handlers today because they contain container
logs, exec, and attach.  I have trouble believing we would define a
conformant Kube cluster as one that doesn't have pod log access.  We need
to rename that bool, it's very deceptive.

On Sun, Feb 7, 2016 at 3:03 PM, David Oppenheimer <notifications@github.com>
wrote:

> In pkg/util/varz/varz.go
> <https://github.com/kubernetes/kubernetes/pull/20528#discussion_r52123666>
> :
>
> > +Copyright 2015 The Kubernetes Authors All rights reserved.
> > +
> > +Licensed under the Apache License, Version 2.0 (the "License");
> > +you may not use this file except in compliance with the License.
> > +You may obtain a copy of the License at
> > +
> > +    http://www.apache.org/licenses/LICENSE-2.0
> > +
> > +Unless required by applicable law or agreed to in writing, software
> > +distributed under the License is distributed on an "AS IS" BASIS,
> > +WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +See the License for the specific language governing permissions and
> > +limitations under the License.
> > +*/
> > +
> > +package varz
>
> configz sounds good.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/20528/files#r52123666>.
>